src/commonMain/kotlin/exchange.dydx.abacus/output/Market.kt

```diff
@@ -550,7 +551,7 @@ data class MarketTrade(
     val side: OrderSide,
     val size: Double,
     val price: Double,
-    val liquidation: Boolean,
```

Comment:
This would make it a breaking change, right?  Should we update the major version?